tests/Assets/Projects/Animals/lib/Badger.php.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="UTF-8">
+  <title>Code Coverage for /var/lib/Badger.php</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ </head>
+ <body>
+  <header>
+   <div class="container-fluid">
```

Comment:
This could be shortened (or even an empty file) if you prefer

Reply:
Or maybe a line of text suggesting it's reason and referring to the test?

Reply:
How's this?
```html
<p>This file emulates a coverage report to ensure only PHP files are found by ClassSearch.</p>
<p>See Phpactor\Tests\System\Extension\SourceCodeFilesystem\Command\ClassSearchCommandTest</p>

<p>Context: https://github.com/phpactor/phpactor/issues/1216</p>
```

I added the `<p>` tags just for the sake of formatting in case someone happens to double-click and it opens in a browser

Reply:
yes that's fine :)